### PR TITLE
have 03 report space usage at end of run

### PR DESCRIPTION
**Pull request by CJ-Wright**
Add diagnostics for why we are failing out on files.

### CODE REVIEW

File: 03.xsh [hunk 1]
```diff
@@ -10,3 +10,9 @@ for i in stages:
     print('FINISHED STAGE {} IN {} SECONDS'.format(i, time.time() - start))
     start = time.time()
 doctr deploy --token --built-docs . --deploy-repo regro/cf-graph --deploy-branch-name master .
+du -hs * | sort -h
+du -hs /tmp/* | sort -h
+import os
+for dir in g`/tmp/`:
```
Comment (scopatz) [inline 03.xsh]: Do you need to have * at the end of this glob?

File: 03.xsh [hunk 2]
```diff
@@ -10,3 +10,9 @@ for i in stages:
     print('FINISHED STAGE {} IN {} SECONDS'.format(i, time.time() - start))
     start = time.time()
 doctr deploy --token --built-docs . --deploy-repo regro/cf-graph --deploy-branch-name master .
+du -hs * | sort -h
+du -hs /tmp/* | sort -h
+import os
```
Comment (scopatz) [inline 03.xsh]: Imports should probably be at the top

File: 03.xsh [hunk 3]
```diff
@@ -10,3 +10,9 @@ for i in stages:
     print('FINISHED STAGE {} IN {} SECONDS'.format(i, time.time() - start))
     start = time.time()
 doctr deploy --token --built-docs . --deploy-repo regro/cf-graph --deploy-branch-name master .
+du -hs * | sort -h
```
Comment (justcalamari) [inline 03.xsh]: Can you use `sort -hr` so the largest files are at the top?

Comment (justcalamari) [reply 03.xsh]: Also can you print the directory name before every `du ...` line?

File: 03.xsh [hunk 4]
```diff
@@ -10,3 +11,8 @@ for i in stages:
     print('FINISHED STAGE {} IN {} SECONDS'.format(i, time.time() - start))
     start = time.time()
 doctr deploy --token --built-docs . --deploy-repo regro/cf-graph --deploy-branch-name master .
+du -hs * | sort -hr
+du -hs /tmp/* | sort -hr
```
Comment (justcalamari) [inline 03.xsh]: Can you print the directory before each `du ...` line?

**Pull request merged**